(CAT-1345) Further update gem README.md (#62)
puppetlabs/puppet-modulebuilder: README.md

As part of this process any non-deliverable aspects of the module, parts of it related to the modules development or testing for example, are stripped away leaving only the documentation and the puppet/ruby code that is needed for the module to function.

> The parts of the module to be excluded are decided by checking for a `.pdkignore`, `.pmtignore` or `.gitignore` file and excluding and excluding any directories or files that are listed in the first one to be found, allowing the user to customize what is and what is not excluded.

Reviewer: ```suggestion
The parts of the module to be excluded are decided are defined in a `.pdkignore`, `.pmtignore` or `.gitignore` file. Any directories or files that are listed in the first one to be found are excluded, allowing the user to customize what is and what is not excluded.
```
